browser/prefs/persistant_pref_names.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_PREFS_PERSISTANT_PREF_NAMES_H_
+#define BRAVE_BROWSER_PREFS_PERSISTANT_PREF_NAMES_H_
+
+#include "build/build_config.h"
+#include "chrome/common/pref_names.h"
+
+const char* const kBravePersistentPrefNames[] = {
```

Comment:
this should be `extern`, the actual list should be in `.cc` file.

Reply:
Hmm, I think this definition should be back to `chromium_src\chrome\browser\prefs\pref_service_incognito_allowlist.cc` instead of using separated files here.
std::size() needs it's definition in compile time.

Reply:
just declare it like so: `const std::vector<const char*>& GetBravePersistentPrefNames();`
and we'll be fine. We don't want to as chromium_src reviewers each time this array is changed.